ouroboros-network/test/Test/Ouroboros/Network/Diffusion/Node.hs

```diff
@@ -201,7 +200,7 @@ run blockGeneratorArgs limits ni na =
                      muxErrorRethrowPolicy
                   <> ioErrorRethrowPolicy
                   <> mkRethrowPolicy
-                      ( \ _ (_ :: CloseError) -> ShutdownNode)
+                      ( \ _ (_ :: SomeException) -> ShutdownNode)
```

Comment:
This should be removed rather than to use `SomeException`.  Even in testing we don't want to shutdown the node if some exception happens (only very specific exceptions).

Note that such a change could be disastrous if committed to the node itself. Why? Because one could try to trigger some erroneous condition in a thread that would result in stopping the whole node.  If one would find a way to propagate this through network, whole system could stop.